README.md

```diff
@@ -91,13 +184,108 @@ You then need to do a second call to the `signIn` mutation providing only the `o
+  - ex: `http://localhost:3000/auth/sorare/callback` (for dev) & `https://myapp.com/auth/sorare/callback` (for prod)
+- the PNG version of your App logo
+
+### OAuth Credentials
+
+Once manually validated, we'll provide you with:
+
+- OAuth UID (or Client ID)
+- OAuth Secret (keep this secret!)
+
+### OAuth Scopes
+
+No specific scopes are available at the moment. All OAuth apps will be have the same privileges.
+
+They are allowed to:
+
+- Access user's nickname, avatar and wallet address
+- Access details about user's Cards, achievements and favourites
+- Access details about user's auctions, offers and notifications
+
+They are NOT allowed to:
+
+- Access user's email address
+- List user's future lineups, compose user's lineups, claim user's rewards
+- Bid on cards, sell user's Cards, make offers, accept offers or initiate a withdrawal
+
+### Access & Refresh Tokens
+
+First you need to create a "Login with Sorare" link in your app and use the following `href`:
+
+```
+https://sorare.com/oauth/authorize?client_id=<YourUID>&redirect_uri=<YourURLEncodedCallbackURI>&response_type=code&scope=
+```
+
+Once signed in Sorare, the user will be asked to authorize your app and will ultimately be redirected to your `redirect_uri` with a `?code=` query parameter, for instance `https://myapp.com/auth/sorare/callback?code=<YourCode>`.
+
+To request an OAuth access token you can then call the `https://api.sorare.com/oauth/token` endpoint with the following parameters:
+
+- `client_id=<YourOAuthUID>`
+- `client_secret=<YourOAuthSecret>`
+- `code=<TheRetrievedCode>`
+- `grant_type=authorization_code`
+- `redirect_uri=<TheSameCallbackURIAsBefore>`
+
+**Example:**
+
+```bash
+$ curl -X POST "https://api.sorare.com/oauth/token" \
+-H 'content-type: application/x-www-form-urlencoded' \
+-d 'client_id=<YourOAuthUID>&client_secret=<YourOAuthSecret>&code=<TheRetrievedCode>&grant_type=authorization_code&redirect_uri=<TheSameCallbackURIAsBefore>'
+
+{"access_token":"....","token_type":"Bearer","expires_in":7200,"refresh_token":"...","scope":"public","created_at":1639608238}
+```
 
-If you want to make requests on behalf of other users you can pass an OAuth access token. We'll need to create the OAuth Application for you first. This is done manually for now, on request.
+You can then use the `access_token` like you would use a JWT token:
+
+```bash
+curl 'https://api.sorare.com/graphql' \
+-H 'content-type: application/json' \
+-H 'Authorization: Bearer <TheUserAccessToken>' \
+-d '{
+    "operationName": "CurrentUserQuery",
+    "query": "query CurrentUserQuery { currentUser { slug } }"
+}'
+
+{"data":{"currentUser":{"slug":"<ASlug>"}}}
+```
 
 ## Rate limit
 
-The API is rate limited by default. We can provide an API Key on demand that raises the limits. The API key should be passed in an http `APIKEY` header.
+The GraphQL API is rate limited by default. We can provide an extra API Key on demand that raises those limits.
```

Comment:
I feel like we should rework this and a bit and not even talk about our "unauthenticated" API but instead talk about our API key and mention that one can make calls without the API key but it's drastically limited.